pymeasure/instruments/attocube/anc300.py

```diff
@@ -139,34 +139,23 @@ class Axis:
         positive.""",
         validator=strict_range, values=[0, inf], check_set_errors=True)
 
-    def __init__(self, controller, axis):
-        self.axis = str(axis)
-        self.controller = controller
+    def insert_id(self, command):
+        """Insert the channel id in a command replacing `placeholder`.
 
-    def _add_axis_id(self, command):
-        """ add axis id to a command string at the correct position after the
-        initial command, but before a potential value
-
-        :param str command: command string
-        :returns: command string with added axis id
+        Add axis id to a command string at the correct position after the
+        initial command, but before a potential value.
         """
         cmdparts = command.split()
-        cmdparts.insert(1, self.axis)
+        cmdparts.insert(1, self.id)
         return ' '.join(cmdparts)
 
-    def ask(self, command, **kwargs):
-        return self.controller.ask(self._add_axis_id(command), **kwargs)
-
-    def write(self, command, **kwargs):
-        return self.controller.write(self._add_axis_id(command), **kwargs)
-
-    def values(self, command, **kwargs):
-        return self.controller.values(self._add_axis_id(command), **kwargs)
+    def _add_axis_id(self, command):
+        self.insert_id(command)
```

Comment:
This was a private method, so not part of the public API. I think it's fair game to delete `_add_axis_id` without process.

Reply:
Ok